src/DotNetWorker.Core/Context/Features/DefaultInputConversionFeature.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Core.Converters;
+using Microsoft.Azure.Functions.Worker.Core.Converters.Converter;
+
+namespace Microsoft.Azure.Functions.Worker.Context.Features
+{
+    /// <summary>
+    /// Default implementation of <see cref="IInputConversionFeature"/>
+    /// </summary>
+    internal sealed class DefaultInputConversionFeature : IInputConversionFeature
```

Comment:
This is the new feature added. This is being called from DefaultModelBindingFeature (for each parameter conversion)